openbas-front/src/admin/components/simulations/simulation/reports/InjectReportResult.tsx

```diff
@@ -0,0 +1,183 @@
+        findInjectCommentsByInjectId(editInjectId)?.report_inject_comment?.length || 0,
+      );
+    }
+  }, [editInjectId, injectsComments]);
+
+  const saveComment = () => {
+    if (!editInjectId) return;
+    const editedComments = findInjectCommentsByInjectId(editInjectId);
+    if (editedComments) {
+      onCommentSubmit(editedComments);
+      setEditInjectId(null);
+    }
+  };
+
+  const columns = [
+    {
+      label: 'Type',
+      render: (inject: InjectResultDTO) => {
+        return inject.inject_injector_contract
+          ? <InjectorContract variant="list" label={tPick(inject.inject_injector_contract?.injector_contract_labels)} />
+          : <InjectorContract variant="list" label={t('Deleted')} deleted={true} />;
+      },
+    },
+    {
+      label: 'Title',
+      render: (inject: InjectResultDTO) => inject.inject_title,
+    },
+    {
+      label: 'Execution date',
+      render: (inject: InjectResultDTO) => <>{fldt(inject.inject_status?.tracking_sent_date)}</>,
+    },
+    {
+      label: 'Scores',
+      render: (inject: InjectResultDTO) => <AtomicTestingResult expectations={inject.inject_expectation_results} />,
+    },
+    {
+      label: 'Targets',
+      render: (inject: InjectResultDTO) => <ItemTargets targets={inject.inject_targets} />,
+    },
+    {
+      label: 'Comments',
+      render: (inject: InjectResultDTO) => {
+        const currentInjectComment = findInjectCommentsByInjectId(inject.inject_id);
+
+        return (editInjectId === inject.inject_id
+          ? <TextField
+              inputRef={inputRef}
+              value={currentInjectComment?.report_inject_comment}
+              onChange={(value: React.ChangeEvent<HTMLInputElement>) => handleCommentChange(currentInjectComment ?? null, value)}
+              multiline
+              variant="outlined"
+              fullWidth
+              autoFocus
+              onBlur={saveComment}
+              sx={{
+                '& .MuiOutlinedInput-root': {
+                  border: 'none', // Removes the border
+                  outline: 'none', // Removes the outline
+                  boxShadow: 'none', // Removes any shadow
+                  padding: '0px', // Adjust padding for textarea
+                },
+                '& .MuiOutlinedInput-notchedOutline': {
+                  border: 'none', // Removes the notched outline
+                },
+                '& .MuiInputBase-input': {
+                  fontWeight: 400,
+                  lineHeight: '1.43',
+                  fontSize: '0.8rem',
+                  padding: '0', // Optional: Remove padding inside input
+                },
+              }}
+            /> : currentInjectComment?.report_inject_comment);
+      },
+    },
+  ];
+
+  return (
+    <div style={style}>
+      <Typography variant="h4" gutterBottom>
+        {t('Injects results')}
+      </Typography>
+
+      <TableContainer component={Paper}>
```

Comment:
I am not sure this is the UI we want
The background and the border are differents
![image](https://github.com/user-attachments/assets/f095ceed-b30c-4da0-a03e-7e56504e0847)